### ISSUE TITLE
Docs and API do not reflect argocd cli actions

### ISSUE THREAD

**Issue by felix-dpg** [argoproj/argo-cd#11018]
**Describe the bug**

In argocd commands docs here:
https://github.com/argoproj/argo-cd/blob/master/docs/user-guide/commands/argocd_app_delete.md

The doc say that you can delete app by label:

```
# Delete apps by label
  argocd app delete -l app.kubernetes.io/instance=my-app
  argocd app delete -l app.kubernetes.io/instance!=my-app
  argocd app delete -l app.kubernetes.io/instance
  argocd app delete -l '!app.kubernetes.io/instance'
  argocd app delete -l 'app.kubernetes.io/instance notin (my-app,other-app)'
```
But argocd app does not have flag **-l** to accomplish that:

```
bash-5.1# argocd app delete -l app=pms --insecure --server X.X.X.X --auth-token ${arg_token}
**Error: unknown shorthand flag: 'l' in -l**
```
Also in api docs (swagger) the flags are not described as possible parameter

![image](https://user-images.githubusercontent.com/72499274/197144738-586d07c7-9dad-4e74-9ef2-17a775537e9e.png)

**Expected behavior**

 The docs need to be synchronized with the possible inputs of argocd command.

**Version**
```
bash-5.1# argocd version --insecure --server X.X.X.X --auth-token ${arg_token}
argocd: v2.2.15+b262249
  BuildDate: 2022-10-18T21:57:16Z
  GitCommit: b26224950abcd2a15d2f94c9adce941f4a9c2375
  GitTreeState: clean
  GoVersion: go1.16.11
  Compiler: gc
  Platform: linux/amd64
argocd-server: v2.4.12+41f54aa
  BuildDate: 2022-09-16T00:45:01Z
  GitCommit: 41f54aa556f3ffb3fa4cf93d784fb7d30c15041c
  GitTreeState: clean
  GoVersion: go1.18.6
  Compiler: gc
  Platform: linux/amd64
  Kustomize Version: v4.4.1 2021-11-11T23:36:27Z
  Helm Version: v3.8.1+g5cb9af4
  Kubectl Version: v0.23.1
  Jsonnet Version: v0.18.0
```

Thanks in advance.

**Comment by maheshbaliga** [comment]
@felix-dpg You are running the command on an older version of the ArgoCD CLI, which does not include this functionality. Kindly update the same to the latest version from [here.](https://argo-cd.readthedocs.io/en/stable/cli_installation/)

**Comment by felix-dpg** [comment]
I open this issue 26 days ago, so at this time the version available was v2.2.15. Version v2.2.16 was released since 14 days ago. Thanks for your reply but your comment is comming too late.

**Comment by felix-dpg** [comment]
Solved in later version.
